.eslintrc.yaml

```diff
@@ -135,16 +135,17 @@ rules:
   }]
   no-tabs: error
   no-trailing-spaces: error
+  object-curly-spacing: [error, always]
```

Comment:
99% of these changes were made using `eslint --fix` automatically. Except for ~30 cases where the insertion of spaces then caused the line to grow longer than 80 characters, which I cleaned up manually following my own stylistic judgement and ensuring eslint passes.